apps/client/src/layout/layout.tsx

```diff
@@ -18,14 +18,15 @@ export function Layout(props: React.PropsWithChildren) {
   const format = useFormat();
   const fullWidth = useFullWidth();
   const { pathname } = useLocation();
+  const params = useParams<{ dashboardId: string }>();
 
-  const isDashboard = pathname === DASHBOARDS_HREF;
+  const hideBreadcrumb = pathname === DASHBOARDS_HREF || !!params.dashboardId;
```

Comment:
you can probably remove `pathname === DASHBOARDS_HREF`  ?

Reply:
If we remove it, breadcrumb will be enabled on the home screen, is it fine?